server/src/transactions/dto/get-transactions-filter.dto.ts

```diff
@@ -0,0 +1,32 @@
+import { IsOptional, IsString, IsNumber, IsDate } from 'class-validator';
+import { TransactionStatus } from '../transaction-status.enum';
+import { User } from 'src/users/entities/user.entity';
+import { Organization } from 'src/organizations/entities/organization.entity';
+import { Asset } from 'src/assets/entities/asset.entity';
+
+export class GetTransactionsFilterDto {
+  @IsOptional()
+  @IsString()
+  status: TransactionStatus;
+
+  // unclear as to the value/conflict of having both @IsOptional and ?
```

Comment:
could be interesting to see if we can figure this out
if you don't find anything online you could `npm run build` then look at the corresponding file in `server/dist/...` to see what it compiles to, make the change and rebuild and see if there is a difference in the file